webtau-groovy/src/main/groovy/com/twosigma/webtau/cli/WebTauCliApp.groovy

```diff
@@ -198,7 +198,7 @@ class WebTauCliApp implements StandaloneTestListener, ReportGenerator {
     @Override
     void generate(Report report) {
         def summary = report.createSummary()
-        problemCount = (int) (summary.failed + summary.errored + summary.skipped)
+        problemCount = (int) (summary.failed + summary.errored)
```

Comment:
Hmmm, there was benefit to including skipped scenarios in the problem count as it would catch someone accidentally pushing a `sscenario`.  Any thoughts on way to reintroduce that capability?

I understand why this change was required here and I'm not too keen on introducing another result type so wondering whether we can do something else that would prevent pushing `sscenario`s.